modules/storages/app/common/storages/peripherals/storage_interaction/authentication_strategies/oauth_user_token.rb

```diff
@@ -124,6 +124,35 @@ def update_refreshed_token(token, http_session)
             token.update(access_token:, refresh_token:)
           end
 
+          def log_and_destroy_token(token, log_message)
```

Comment:
This method seems unused - you probably planned on using it in `handle_http_error_on_refresh` and `handle_timeout_on_refresh`?

Reply:
indeed, will delete it.